core/network/src/main/kotlin/com/google/samples/apps/nowinandroid/core/network/model/NetworkNewsResource.kt

```diff
@@ -35,6 +35,20 @@ data class NetworkNewsResource(
     val topics: List<String> = listOf(),
 )
 
+fun NetworkNewsResource.asExternalModel(topics: List<NetworkTopic>) =
```

Comment:
This looks very similar to `NetworkNewsResourceExpanded`, where there is a list of `NetworkTopic`s. Is there a good way to combine those representations?

Reply:
`NetworkNewsResourceExpanded` isn't actually used anywhere except for tests so it can be safely deleted. 

I believe it was added because there's an endpoint on the backend server that will supply data in this format, we just don't use it.